python/graphstorm/sagemaker/sagemaker_train.py

```diff
@@ -164,7 +164,16 @@ def run_train(args, unknownargs):
         os.makedirs(restore_model_path, exist_ok=True)
     else:
         restore_model_path = None
-    output_path = "/tmp/gsgnn_model/"
+
+    if args.model_artifact_s3:
```

Comment:
Maybe we should add an assertion here.

Reply:
An assertion for the value of args.model_artifact_s3? Like `assert args.model_artifact_s3.startswith("s3://")` ?